src/test/resources/features/WorkloadServiceImplTest.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Workload management
+
+  Scenario: Add workload for a trainer
+    Given the database contains a trainer with username "john_doe" and the following yearly summaries:
+      | year | month    | workAmount |
+      | 2023 | JANUARY  | 120        |
+    When the workload is added with the following details:
+      | username | date       | duration | actionType |
+      | john_doe | 2023-01-15 | 60       | ADD        |
```

Comment:
to make test stronger here could be more data relative to another months to check that nothing is mixed from another months